src/bundles/CaseStudy/redux/modules/errorMessage.js

```diff
@@ -0,0 +1,21 @@
+const MESSAGE = 'error/message'
+const REMOVE = 'error/remove'
+
+export default function reducer(state = {}, action = {}) {
+  const { type, key, value } = action
+  switch (type) {
+    case MESSAGE:
+      state[key] = value;
```

Comment:
Yep sure does - I've addressed it in next PR.

This whole module is awful, needs to be ripped out/moved